Remove libgio from AppImage

TheTumultuousUnicornOfDarkness opened the pull request:
Hello,

I have issues with AppImage on Arch Linux and Fedora 33:
```
symbol lookup error: /usr/lib/libpangoft2-1.0.so.0: undefined symbol: g_task_set_name
```
There is an incompatibility with GIO (bundled in AppImage) and Cairo (system library). This issue seems to affect other libraries like GVFS.

Removing libgio.so from AppImage does not seem to introduce regression on Ubuntu 16.04 and 20.04, and it fixes issue on Arch Linux and Fedora 33.

TheTumultuousUnicornOfDarkness committed:
Remove libgio from AppImage

g_task_set_name symbol is missing on older versions but required by system libraries

TheAssassin reviewed (commented):
Please either remove all the lines or comment them out. Removing one and keeping the other doesn't make a lot of sense.

TheTumultuousUnicornOfDarkness commented:
Indeed. Fixed.

TheTumultuousUnicornOfDarkness committed:
Fix for CI

In /home/runner/work/linuxdeploy-plugin-gtk/linuxdeploy-plugin-gtk/linuxdeploy-plugin-gtk.sh line 94:
if ! which patchelf &>/dev/null && ! type patchelf &>/dev/null; then
     ^---^ SC2230: which is non-standard. Use builtin 'command -v' instead.

TheAssassin commented:
Thank you very much!
